crates/rpc/src/server/mod.rs

```diff
@@ -78,6 +80,7 @@ impl Rpc {
             .layer(TraceLayer::new_for_grpc().make_span_with(traced_span_fn(TracedComponent::Rpc)))
             .layer(AcceptHeaderLayer::new(&rpc_version, genesis.commitment()))
             .layer(cors_for_grpc_web_layer())
+            .layer(HealthCheckLayer)
```

Comment:
I think we should also swap the two layers above maybe? This is not related to this PR, but I'm worried that if the `Accept` layer rejects the request, then the CORS related headers won't be applied and a web client wouldn't accept the response at all, obscuring the error.

Is that how it works?

Reply:
Yea, done. LMK if thats what you meant.

Reply:
Perfect, yeah I think that's right.